errors.md

```diff
@@ -3,13 +3,14 @@
 ## Use code 1000s for SDK
 
 | Code | Description                                                                                                 | Code Path                                           |
-|------|-------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
+| ---- | ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------- |
 |      | 1000-1099 - for general agent errors                                                                        |                                                     |
 | 1000 | Unsupported agent type                                                                                      | pkg/agent/ErrUnsupportedAgentType                   |
 | 1001 | initialization error checking for dependencies to respond, possibly network or settings                     | pkg/util/errors/ErrInitServicesNotReady             |
 | 1002 | timeout error checking for dependencies to respond, possibly network or settings                            | pkg/util/errors/ErrTimeoutServicesNotReady          |
 | 1003 | periodic health checker or status updater failed.  Services are not ready                                   | pkg/util/ErrPeriodicCheck                           |
 | 1004 | error starting periodic status update                                                                       | pkg/util/ErrStartingPeriodicStatusUpdate            |
+| 1005 | error starting immediate status update                                                                      | pkg/util/ErrStartingImmediateStatusUpdate           |
```

Comment:
I feel like 1004 and 1005 can be combined into one error code with a string arg

Reply:
#true story... will update

Reply:
updated